versioned_docs/version-v7/theming/advanced.md

```diff
@@ -168,7 +168,7 @@ After running through the Sass compiler, the colors will have the following valu
 
 | Variable            | Value                                          |
 | ------------------- | ---------------------------------------------- |
-| `$background`       | <CodeColor color="#3880ff">#3880ff</CodeColor> |
+| `$background`       | <CodeColor color="#0054e9">#0054e9</CodeColor> |
 | `$background-shade` | <CodeColor color="#3171e0">#3171e0</CodeColor> |
 | `$background-tint`  | <CodeColor color="#4c8dff">#4c8dff</CodeColor> |
```

Comment:
tint & shade should be adjusted too, i think, using the `mix` functions above this:

```suggestion
| `$background`       | <CodeColor color="#0054e9">#0054e9</CodeColor> |
| `$background-shade` | <CodeColor color="#004acd">#004acd</CodeColor> |
| `$background-tint`  | <CodeColor color="#1a65eb">#1a65eb </CodeColor> |
```